sonic-net/sonic-buildimage
# [sflow] Switch to bullseye

Change references to use bullseye instead of buster

#### Why I did it

 * Almost all daemons in 202211 and master uses bullseye, and sflow was easy to migrate.

#### How I did it

 * Replaced the references, built and tested in 202211.

#### How to verify it

 * Build with the changes, enable sflow:
```
admin@sonic:~$ sudo config sflow collector add test 1.2.3.4
admin@sonic:~$ sudo config sflow collector enable
```
 * tcpdump on `1.2.3.4` and see that UDP sFlow are being sent.

#### Which release branch to backport (provide reason below if selected)


I am packaging 202211 for use in our production and would like this upgrade in that release if possible.

#### Description for the changelog

 * sflow feature is now based on Bullseye

#### A picture of a cute animal (not mandatory but encouraged)

![image](https://user-images.githubusercontent.com/149442/229363971-6e26a6fc-c96f-4d50-aa4f-a478137ce16c.png)
